src/sentry/sentry_metrics/indexer/mock.py

```diff
@@ -0,0 +1,71 @@
+from typing import List, Optional, Tuple
+
+from sentry.models import Project
+
+from .base import StringIndexer, UseCase
+
+_STRINGS = {
+    "abnormal": 0,
```

Comment:
do we want those numbers to be the respective FMV-hash already? I fear that if we ever switch to regular string indexer we will have to special-case those values forever

Reply:
We switched to a regular string indexer. We are not planning to use hashes anymore. The reason is that, in order to have a good confidence of no collisions, we had to partition the index in ways that made too many assumptions about how the product works that we cannot verify now.